<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtOpenGLDemoClass</class>
 <widget class="QWidget" name="QtOpenGLDemoClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtOpenGLDemo</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>30</y>
     <width>161</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Projection</string>
   </property>
   <widget class="QRadioButton" name="originalButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>132</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Original</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="invertButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>132</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Invert Filter</string>
    </property>
   </widget>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>390</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>360</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Collison</string>
   </property>
  </widget>
  <widget class="QPushButton" name="configButton">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>150</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Configure File</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_2">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>220</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>180</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Directly</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtOpenGLDemo.qrc"/>
 </resources>
 <connections/>
</ui>
